types/ramda/package.json

```diff
@@ -0,0 +1,9 @@
+{
```

Comment:
`package.json` isn't meant for this purpose. You shouldn't need a `package.json` at all.